### PR TITLE
improve docs for supported operators in Variable

### PR DESCRIPTION
**Pull request by kmaehashi**
After the offline discussion with @hvy regarding #4495, I thought it is better to explicitly state what operators are supported for Variables.

### CODE REVIEW

File: chainer/variable.py [hunk 1]
```diff
@@ -432,6 +434,23 @@ class Variable(object):
...
+        * Subtraction: ``a - b`` (:meth:`__sub__`, :meth:`__rsub__`)
+        * Multiplication: ``a * b`` (:meth:`__mul__`, :meth:`__rmul__`)
+        * Division: ``a / b`` (:meth:`__div__`, :meth:`__rdiv__`, \
+                               :meth:`__truediv__`, :meth:`__rtruediv__`)
+        * Floor Division: ``a // b`` (:meth:`__floordiv__`, \
+                                      :meth:`__rfloordiv__`)
+        * Exponentiation: ``a ** b`` (:meth:`__pow__`, :meth:`__rpow__`)
+        * Matirx Multiplication: ``a @ b`` (:meth:`__matmul__`, \
+                                            :meth:`__rmatmul__`)
+        * Negation (Arithmetic): ``- a`` (:meth:`__neg__`)
```
Comment (hvy) [inline chainer/variable.py]: How about `-a`, removing the whitespace?

**Comment by hvy** [comment]
Do you think `abs` should be added to the list as well?

File: chainer/variable.py (hunk 1 above)
Comment (kmaehashi) [reply chainer/variable.py]: I followed expressions used in https://docs.python.org/3/library/operator.html#mapping-operators-to-functions.

**Comment by kmaehashi** [comment]
`abs` is not an operator, so I think just documenting `__abs__` is enough.
(I feel it is better to encourage use of `F.abs` rather than `abs` for consistency.)

**Comment by hvy** [comment]
I see, thanks for the clarification. Then let's leave it as is.

**Comment by hvy** [comment]
jenkins, test this please.

**Comment by hvy** [comment]
jenkins, test this please.

**Comment by hvy** [comment]
jenkins, test this please.

**Comment by hvy** [comment]
LGTM!

**Pull request merged**